### PR TITLE
Fix stack overflow

### PR DESCRIPTION
**Pull request by kbirk**

### CODE REVIEW

**Review by YohannParis — APPROVED** [review]
LGTM

File: packages/client/hmi-client/src/types/Types.ts [hunk 1]
```diff
@@ -688,7 +688,6 @@ export interface CiemssStatusUpdate {
     loss: number;
     progress: number;
     jobId: string;
-    dataToPersist: any;
```
Comment (dvince2) [inline packages/client/hmi-client/src/types/Types.ts:691]: If these are super large do we want them passed up to the front end as well or should they be hit with a `@TSIgnore`? (I'm not sure the context here)

**Review by dvince2 — APPROVED** [review]

File: packages/client/hmi-client/src/types/Types.ts (hunk 1 above)
Comment (kbirk) [reply packages/client/hmi-client/src/types/Types.ts:691]: Good catch. Fixing

Comment (kbirk) [reply packages/client/hmi-client/src/types/Types.ts:691]: Wait nvm the ignore already covered it.

Comment (dvince2) [reply packages/client/hmi-client/src/types/Types.ts:691]: We still probably want it on here then that way the field doesn't show up on the front end, even if its empty

**Pull request merged**